Server/ConcordiaCurriculumManager/Services/CourseService.cs

```diff
@@ -92,15 +105,25 @@ public async Task<CourseCreationDossier> InitiateCourseCreation(CourseCreationIn
         }
         _logger.LogInformation($"Inserted ${typeof(Course)} ${course.Id} by {typeof(User)} ${user.Id}");
 
-        var dossier = new CourseCreationDossier { Id = new Guid(), InitiatorId = user.Id, NewCourseId = course.Id };
-        bool dossierCreated = await _dossierRepository.SaveCourseCreationDossier(dossier);
-        if (!dossierCreated)
+        var courseCreationRequest = new CourseCreationRequest { Id = new Guid(), InitiatorId = user.Id, NewCourseId = course.Id, Dossier = dossier };
```

Comment:
Can you use `Guid.NewGuid()` instead of `new Guid()`, which I've realised recently is not the correct way to do it.